agent/api/task.go

```diff
@@ -145,6 +145,8 @@ type Task struct {
 
 	// lock is for protecting all fields in the task struct
 	lock sync.RWMutex
+
+	cpuUnboundedWindowsEnabled bool
```

Comment:
`cpuUnboundedWindowsEnabled` is quite a long name. Can you make it shorter? `cpuUnboundedWindows` perhaps? `Enabled` seems redundant since its already a `bool`.

Also, the right thing to do here is to probably create `Task` struct in `task_windows.go` and make this a part of that struct. But, we can do that in a separate PR.